storage/s3/src/integration-test/java/io/aiven/kafka/tieredstorage/storage/s3/S3StorageTest.java

```diff
@@ -50,8 +50,8 @@ public class S3StorageTest extends BaseStorageTest {
 
     @BeforeAll
     static void setUpClass() {
-        final var clientBuilder = S3Client.builder();
-        clientBuilder.region(Region.of(LOCALSTACK.getRegion()))
+        s3Client = S3Client.builder()
```

Comment:
nit: it probably doesn't matter for a test, but i think that `s3Client` needs to be closed in a `@AfterAll` method to cleanup eventual connections/resources.